doc/1.manual/x5.htm

```diff
@@ -25,8 +25,9 @@ <H3> <A name="s2"> 5.1. release notes </A> </H3>
 <P> More bug fixes.  Audio on linux improved; various improvements to the
 pd~ object, -no-gui windows patches able to use netsend/netreceive; loadbang
 order fix for clone object; soundfiler no longer limiting size of soundfiles
-to 4million points by default; updates to deken (external extension search
-tool); many code-level enhancements.
+to 4million points by default; inlet of noise~ now sets the seed instead of 
+doing nothing supdates to deken (external extension searchtool); 
```

Comment:
that's a typo: "doing nothing supdates"... giving the size and nature of this commit, i'd say this is a major issue :-)

it would also be nice to have meaningful commit messages: "Update x5.htm` is pretty meaningless (and redundant, as this information is already stored in the commit itself)

Reply:
did it by hand.